rskj-core/src/main/java/co/rsk/net/sync/SyncConfiguration.java

```diff
@@ -45,6 +50,8 @@ public final class SyncConfiguration {
     private final Duration timeoutWaitingSnapChunk;
 
     private final int snapshotSyncLimit;
+    private final List<Node> snapshotTrustedPeers;
```

Comment:
looks like we do not need this field (as well as `getSnapshotTrustedPeers()`), as it's only being used in the ctor

Reply:
solved, removed `getSnapshotTrustedPeers()`. We pass the snapshotTrustedPeers in the SyncConfiguration constructor in the RSKContext class:
```
protected synchronized SyncConfiguration buildSyncConfiguration() {
        checkIfNotClosed();

        RskSystemProperties rskSystemProperties = getRskSystemProperties();
        return new SyncConfiguration(
                rskSystemProperties.getExpectedPeers(),
                rskSystemProperties.getTimeoutWaitingPeers(),
                rskSystemProperties.getTimeoutWaitingRequest(),
                rskSystemProperties.getExpirationTimePeerStatus(),
                rskSystemProperties.getMaxSkeletonChunks(),
                rskSystemProperties.getChunkSize(),
                rskSystemProperties.getMaxRequestedBodies(),
                rskSystemProperties.getLongSyncLimit(),
                rskSystemProperties.getTopBest(),
                rskSystemProperties.isServerSnapshotSyncEnabled(),
                rskSystemProperties.isClientSnapshotSyncEnabled(),
                rskSystemProperties.getSnapshotChunkTimeout(),
                rskSystemProperties.getSnapshotSyncLimit(),
                rskSystemProperties.getSnapshotTrustedPeers());
    }
```

Reply:
yeah, we do, but inside `SyncConfiguration` the `snapshotTrustedPeers` field is not used anywhere and can be removed

Reply:
addressed, removed the `snapshotTrustedPeers` field